02/Evgeny Vasiukevich/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ChangeEvent, useState } from 'react';
+import HotelsBlock from './components/HomesGuestsLoves/HotelsBlock';
+import SearchForm from './components/UI/SearchForm/SearchForm';
+import { hotelsData } from './data/hotelsData';
+
+function App() {
+  const [filterHotels, setFilterHotels] = useState(hotelsData);
```

Comment:
probably would it better to put all of this code to separate component (not in the App)?

Reply:
In this hw I did your advice but in the next hw I need again to add states in the App to run states between Search and HotelsBlock.
Your idea is cool and I'm going to do it in a next homework to join all components in one. Could you give me some advice how this general component could be called?